### ISSUE TITLE
hostctl adds entries for the same domain + ip

### ISSUE THREAD

**Issue by ddl-jbrown** [guumaster/hostctl#71]
**Describe the bug**
hostctl appears to not be idempotent when it adds entries for the same domain + ip when run multiple times.

**To Reproduce**
Run this command multiple times (`--uniq` doesn't seem to make a difference)
```
touch /tmp/test-hosts; hostctl add domains test domain1 --ip 127.0.0.1 --host-file /tmp/test-hosts
```

Current behavior
```
##################################################################
# Content under this line is handled by hostctl. DO NOT EDIT.
##################################################################

# profile.on test
127.0.0.1 domain1
127.0.0.1 domain1
127.0.0.1 domain1
127.0.0.1 domain1
# end
```

**Expected behavior**
When running the command above, `/tmp/test-hosts` should only contain one uniq entry.

Example of what it should look like
```
##################################################################
# Content under this line is handled by hostctl. DO NOT EDIT.
##################################################################

# profile.on test
127.0.0.1 domain1
# end
```

**System (please complete the following information):**
 - OS: Mac
 - Version: 10.15.7 (19H15)

**Additional context**
```
hostctl --version
hostctl version 1.0.14
```

**Comment by lukasmrtvy** [comment]
any progress on this ? thanks

**Comment by guumaster** [comment]
@lukasmrtvy thanks for bringing up this issue. I'm working on a solution with the `--uniq` param. But I'm thinking if it should always treat the pairs domain/ip as unique per profile as default. What do you think?

**Comment by guumaster** [comment]
Fixed on release v1.1.0. Thanks for the feedback!
